manager/integration/tests/test_base_image.py

```diff
@@ -0,0 +1,203 @@
+import time
+import common
+
+from common import clients, volume_name  # NOQA
+from common import SIZE, DEV_PATH, BASE_IMAGE
+from common import wait_for_volume_state, wait_for_volume_delete
+from common import generate_volume_name
+from test_basic import snapshot_test, backup_test
+
+
+def test_volume_basic(clients, volume_name):  # NOQA
```

Comment:
Can you refactor `test_basic.py` to avoid duplicated code? E.g. pass the BASE_IMAGE as a string parameter to the function and the comparison in the volume properties check would still work.